### PR TITLE
Update table

### PR DESCRIPTION
**Pull request by jaamanu**
In this PR 
Add a column species of type string to your animals table. Modify your schema.sql file.
Insert the following data:
Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times
add these insertions to your data.sql file.

Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.
Verify that change was made and persists after commit.
Take a screenshot of the results of your actions.
Screenshot 2022-12-16 at 12 19 07

Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Take a screenshot of the results of your actions.
Remember to add all these transactions to your queries.sql file.
Screenshot 2022-12-16 at 12 20 52

Write queries to answer the following questions:
How many animals are there?
How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Take a screenshot of the results of your queries.
add these queries to your queries.sql file.
add screenshots with the results of your queries and with the results of your play with transactions to your Pull Request description.
Screenshot 2022-12-16 at 12 23 03

Update screen shot 
Screenshot 2022-12-16 at 13 34 57

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (3, 'Pikachu', 'Jan 07, 2021', 1, false, 15.04);
 
-INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES (4, 'Devimon', 'May 12, 2017', 5, true, 11);
+
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (5, 'Charmander', 'Feb 03, 2020', 0, false, 11);
```
Comment (AnselemOdims) [inline data.sql]: - [ ] Good job on the queries but please kindly take note that the weight for this query is `-11kg`. Kindly ensure your queries stick to the project guidelines.

File: data.sql [hunk 2]
```diff
@@ -1,17 +1,38 @@
...
+
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (6, 'Plantmon', 'Nov 15, 2021', 2, true, 5.7);
```
Comment (AnselemOdims) [inline data.sql]: - [ ] As stated above,  please kindly take note that the weight for this query is `-5.7kg`. Kindly ensure your queries stick to the project guidelines.

File: data.sql [hunk 3]
```diff
@@ -1,17 +1,38 @@
...
+
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (7, 'Squirtle', 'Jun 12, 2005', 1, true, 45);
```
Comment (AnselemOdims) [inline data.sql]: - [ ] As stated above,  please kindly take note that the weight for this query is `-12.3kg`. Kindly ensure your queries stick to the project guidelines.

File: data.sql [hunk 4]
```diff
@@ -1,17 +1,38 @@
...
+
+INSERT INTO animal (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (8, 'Angemon', 'Jun 12, 2005', 1, true, 45);
```
Comment (AnselemOdims) [inline data.sql]: - [ ] As stated above,  please kindly take note that the weight for this query is `-45kg`. Kindly ensure your queries stick to the project guidelines.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @jaamanu , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @jaamanu,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**